packages/video_player/video_player_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.1
+
+* Updates to  Pigeon 13.
```

Comment:
nit '  ' -> ' '.

I can't believe this got merged...

Reply:
Doh! Luckily it shouldn't show up that way on pub.dev.